web3.js/test/connection.test.ts

```diff
@@ -1442,6 +1623,7 @@ describe('Connection', () => {
         const mintOwner = new Account();
         const accountOwner = new Account();
         const token = await Token.createMint(
+          // @ts-ignore
```

Comment:
The Connection signatures no longer lined up between this Connection version and the web3.js inside spl-token library.

Reply:
Ouch, that means downstream users will likely encounter the same issue if they're using TypeScript. Let's make sure we always make backwards compatible changes to the `Connection` interface